src/js/components/AdvancedConfig.js

```diff
@@ -7,7 +7,7 @@ const METHODS_TO_BIND = [
   'onResize',
   'triggerSubmit'
 ];
-const MOBILE_WIDTH = 480;
+const MOBILE_WIDTH = 768 - 1;
```

Comment:
This is reflecting how styles determine mobile width; by taking small-screen width minus 1.
I did not leave a comment here, since this will be removed as soon as the mobile logic in schema form has been converted to styles.